### PR TITLE
Feature/display error login

### PR DESCRIPTION
**Pull request by BrandonV21**
Added an error message to display to the user, if unable to login. 

image

### CODE REVIEW

File: src/components/users/login/LoginPage.js [hunk 1]
```diff
@@ -32,7 +32,11 @@ const LoginPage = (props) => {
         setLoginResponse(res.data);
         navigate("../posts", { replace: true });
       })
-      .catch((err) => console.log(err.response));
+      .catch((err) => {
+        console.log(err.response)
+        const errorMessage = err.response.data.data.email
+        setLoginError(errorMessage)
+      });
   };
```
Comment (PointWolfAnj) [inline src/components/users/login/LoginPage.js:41]: looks great!

**Review by PointWolfAnj — DISMISSED** [review]

**Review by phillipdunn — APPROVED** [review]
LGTM, just need to sort out cypress before you move

**Review by Auenc — APPROVED** [review]
Looks good. As Phil said, just need to solve the cypress test issues and then this will be good to merge!

**Comment by mikemherron** [comment]
You'll need to resolve this warning befre you can merge: 

> React Hook useEffect has a missing dependency: 'handleFindProfile'. Either include it or remove the dependency array  react-hooks/exhaustive-deps

**Review by mikemherron — APPROVED** [review]

**Review by PointWolfAnj — APPROVED** [review]

**Pull request merged**